aries_cloudagent/core/conductor.py

```diff
@@ -274,6 +274,13 @@ def dispatch_complete(self, message: InboundMessage, completed: CompletedTask):
             LOGGER.exception(
                 "Exception in message handler:", exc_info=completed.exc_info
             )
+            if isinstance(completed.exc_info[1], LedgerConfigError) or isinstance(
+                completed.exc_info[1], LedgerTransactionError
+            ):
+                LOGGER.error("Shutdown with %s", str(completed.exc_info[1]))
```

Comment:
Could cover with unit test in `test_conductor.py`:
```
    async def test_dispatch_complete_fatal_x(self):
        builder: ContextBuilder = StubContextBuilder(self.test_settings)
        conductor = test_module.Conductor(builder)

        message_body = "{}"
        receipt = MessageReceipt(direct_response_mode="snail mail")
        message = InboundMessage(message_body, receipt)
        mock_task = async_mock.MagicMock(
            exc_info=(test_module.LedgerTransactionError, ("Ledger is wobbly"), "..."),
            ident="abc",
            timing={
                "queued": 1234567890,
                "unqueued": 1234567899,
                "started": 1234567901,
                "ended": 1234567999,
            },
        )

        await conductor.setup()
        conductor.dispatch_complete(message, mock_task)
```

Reply:
Test added, thanks @sklump !